distrobox-init

```diff
@@ -415,63 +415,74 @@ if [ "${upgrade}" -ne 0 ] ||
 		if ! apk add "${shell_pkg}"; then
 			shell_pkg="bash"
 		fi
-		deps="
+		if apk add wolfi-base; then
+			deps="
+				busybox
+				gnutar
+				man-db
+				mesa
+				posix-libc-utils
+				sudo-rs
```

Comment:
This package conflicts with `util-linux-login` that is needed for a full `su` package

Either we remove sudo-rs or we solve this upstream 

![image](https://github.com/89luca89/distrobox/assets/598882/c01b386e-41ea-4de0-b61f-a38510015caa)